lms/resources/_js_config/__init__.py

```diff
@@ -189,6 +189,18 @@ def enable_lti_launch_mode(self):
             "allowedOrigins": self._request.registry.settings["rpc_allowed_origins"]
         }
 
+    def _create_assignment_api(self):
+        if not self._context.is_canvas:
+            return None
+
+        return {
+            "path": self._request.route_path("canvas_api.assignments.create"),
+            "data": {
+                "ext_lti_assignment_id": self._request.params["ext_lti_assignment_id"],
+                "course_id": self._request.params["custom_canvas_course_id"],
```

Comment:
We also send `custom_canvas_course_id` to the FE, which in turn will send it back to the backed on create_assignment so we can generate a `canvas://`  url, which includes the course.